main/src/ca/uwaterloo/flix/language/phase/Parser.scala

```diff
@@ -151,7 +151,7 @@ class Parser(val source: Source) extends org.parboiled2.Parser {
 
   def UsesOrImports: Rule1[Seq[ParsedAst.UseOrImport]] = rule {
     // It is important for documentation comments that whitespace is not consumed if no uses are present
-    (optWS ~ oneOrMore((Use | Import) ~ optWS ~ ";").separatedBy(optWS)) | push(Seq.empty)
+    optWS ~ oneOrMore(Use | Import).separatedBy(optWS) | push(Seq.empty)
```

Comment:
I'm not sure what the precedence of ~ and | is, but its should be `(a ~ b) | c` not `a ~ (b | c)` otherwise the docs comments after `use` is consumed. maybe just keep the paranthesis for clarity, whatever the precedence is?

Reply:
Agreed.